AutoCollection/ClientRequestParser.ts

```diff
@@ -54,9 +63,9 @@ class ClientRequestParser extends RequestParser {
         let remoteDependency = new Contracts.RemoteDependencyData();
         remoteDependency.type = Contracts.RemoteDependencyDataConstants.TYPE_HTTP;
 
-        if (this.targetIKeyHash) {
+        if (this.correlationId) {
             remoteDependency.type = Contracts.RemoteDependencyDataConstants.TYPE_AI;
-            remoteDependency.target = urlObject.hostname + " | " + this.targetIKeyHash;
+            remoteDependency.target = urlObject.hostname + " | " + this.correlationId;
```

Comment:
@nizarq is it just this line that should check for a default `cid-v1:` correlationId? Still mark it as `TYPE_AI` since it specified something, but not include the default value in the `target`?